src/utils/getPostTime.js

```diff
@@ -0,0 +1,36 @@
+const getPostTime = postDate => {
```

Comment:
Its good that you did is manually, although, using react-moment or date-fns would be preferred.

Reply:
I haven't explored react-moment yet. Will look into it sometime later.